controllers/inverse_kinematics_pyikfast/inverse_kinematics_pyikfast.py

```diff
@@ -21,19 +21,26 @@
 from pyikfast_module import inverseKinematics
 from spawn_target import spawnTarget
 
-
+from transforms3d.axangles import mat2axangle
 
 # ------------------- CONFIGURATION -------------------------
 
 # how many simulationsteps before calculating the next IK solution.
 # For velocity control, a smaller value makes the movement smoother.
 IKstepSize = 2
 
+supervisor = Supervisor()
+
 # This name is needed in the pyikfast_module.py to determine the joint2_offset (line 25)
-protoName = 'Irb4600-40'
+protoName = supervisor.getSelf().getTypeName()  #'UR10e' #'Irb4600-40'
 
 # import the pyikfast solver module for your robot as "pyikfast_module"
-import pyikfast_irb4600_40 as pyikfast_module
+# import pyikfast_irb4600_40 as pyikfast_module
```

Comment:
This is fine for now, but the comment should be expanded to explain that any custom solver can be imported. It would also be good to add more robots and sample solvers. In that case we should switch to a case structure.